src/components/Shared/Map/CrossingMap.js

```diff
@@ -500,13 +538,23 @@ class CrossingMap extends React.Component {
             coordinates={
               JSON.parse(this.state.selectedCrossing.geojson).coordinates
             }
+            anchor="bottom"
           >
             <div>
               {this.state.selectedCrossing.crossingName}
-              {this.props.mobile && (
-                <button onClick={() => this.props.setShowDetailsOnMobile(true)}>
-                  Details
-                </button>
+              {this.props.mobile &&
+                (!this.state.showDetailsOnMobile &&
+                  this.state.selectedCrossing.crossingStatus !==
+                    STATUS_OPEN) && (
```

Comment:
@Niicck I wasn't able to reproduce your "no details button" issue, I'm assuming it's because you were looking at open crossings. I updated the logic so when we click on an open crossing (no details about closure etc.) we don't show a details button.

I'm not sure this is the best path, as we could definitely have some notes in the details, but it seems like a cleaner overall UX than having a bunch of details buttons that do nothing.

Reply:
You're right - I wasn't looking at a closed crossing. It works fine